pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -1248,3 +1248,49 @@ func TestGetDeploymentsForReplicaSet(t *testing.T) {
 	}
 
 }
+func generateRSWithRevision(revision string) apps.ReplicaSet {
+	return apps.ReplicaSet{
+		ObjectMeta: metav1.ObjectMeta{
+			UID:         randomUID(),
+			Name:        names.SimpleNameGenerator.GenerateName("replicaset"),
+			Labels:      map[string]string{"foo": "bar"},
+			Annotations: map[string]string{RevisionAnnotation: revision},
+		},
+	}
+}
+
+func TestLastRevision(t *testing.T) {
+	rs1 := generateRSWithRevision("1")
+	rs2 := generateRSWithRevision("2")
+	rs3 := generateRSWithRevision("3")
+
+	tests := []struct {
+		name           string
+		replicaSets    []*apps.ReplicaSet
+		expectRevision int64
```

Comment:
nit:
```suggestion
		wantRevision int64
```

it's shorter and more idiomatic in golang.